### PR TITLE
OE-323

### PR DESCRIPTION
**Pull request by apigram**
Practice field now supports searching by practice name as well as address.
Practice name will now be displayed in the practice card on the Add Patient screen.

### CODE REVIEW

File: protected/controllers/PatientController.php [hunk 1]
```diff
@@ -1845,14 +1845,16 @@ public function actionPracticeList($term)
         $criteria->addCondition('( (date_end is NULL OR date_end > NOW()) AND (date_start is NULL OR date_start < NOW()))');
         
         $criteria->addSearchCondition('LOWER(CONCAT_WS(", ", address1, address2, city, county, postcode))', $term);
+        $criteria->addSearchCondition('first_name', $term, true, 'OR');
```
Comment (marshl) [inline protected/controllers/PatientController.php]: Should the first name be LOWER()'d to provide case insensitive matching?

File: protected/controllers/PracticeController.php [hunk 2]
```diff
@@ -74,9 +74,10 @@ public function actionCreate($context = null)
                 $context === 'AJAX');
         }
         if ($context === 'AJAX') {
```
Comment (marshl) [inline protected/controllers/PracticeController.php]: I thought the AJAX context thing was fixed ages ago...

**Review by marshl — APPROVED** [review]

File: protected/controllers/PatientController.php (hunk 1 above)
Comment (apigram) [reply protected/controllers/PatientController.php]: Yes. Good point.

File: protected/controllers/PracticeController.php (hunk 2 above)
Comment (apigram) [reply protected/controllers/PracticeController.php]: I think we said we'd revisit it in Phase 2.

**Review by marshl — APPROVED** [review]

**Pull request merged**